aepsych/database/tables.py

```diff
@@ -368,11 +371,91 @@ def __repr__(self):
         )
 
     @staticmethod
-    def update(engine):
+    def update(server, engine):
```

Comment:
I think `server` here is actually a `Database` object (based on how it is being used), so that's a bit of a confusing argument name.

Reply:
True! Thanks for pointing it out! 🙌🏽